notebooks/official/model_monitoring/model_monitoring.ipynb

```diff
@@ -472,33 +472,16 @@
       },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #38.    ATTRIB_DRIFT_CUSTOM_THRESHOLDS = {"cnt_level_start_quickplay": 0.01}
</pre><p>These custom thresholds aren't being used anywhere in the rest of the notebook. For clarify and conciseness, I would recommend:</p><ol><li>remove the <code>*_CUSTOM_THRESHOLDS</code> variables</li><li>rename <code>SKEW_DEFAULT_THRESHOLDS</code> to <code>SKEW_THRESHOLDS</code>, and do the same for the other threshold variables</li><li>add a note after <code>DEFAULT_THRESHOLD_VALUE = 0.001</code> to encourage users to modify the threshold value and the config keys, so that they can monitor other features in the model as well</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/879/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
Done!